src/components/ResponsiveContent/ResponsiveContent.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import styles from './ResponsiveContent.styl'
+
+const ResponsiveContent = ({ children, className }) => {
+  const responsiveContentClassName = cx(
+    styles.ResponsiveContentSize_default,
+    className
+  )
+
+  return <div className={responsiveContentClassName}>{children}</div>
+}
+
+ResponsiveContent.propTypes = {
+  /**
+   * The content of the element.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * The CSS class name of the element to extend it.
+   */
```

Comment:
Nit: No need for comments when they are self describing.

For short comments I prefer this way, it adds less lines and is more compact.

```
Component.propTypes = {
  /** @type {String} - Name of the person */
  name: PropTypes.string.required
}
```

This is understood by styleguidist.